src/AppInstallerCLICore/Command.cpp

```diff
@@ -291,7 +339,10 @@ namespace AppInstaller::CLI
 
         for (auto& command : commands)
         {
-            if (Utility::CaseInsensitiveEquals(*itr, command->Name()))
+            if (
+                Utility::CaseInsensitiveEquals(*itr, command->Name()) ||
+                Utility::CaseInsensitiveContains(command->Aliases(), *itr)

```

Comment:
I think at some point we send telemetry about what command was executed with its full name. I think it could be helpful to report the specific alias. If most people use the alias than the original command, maybe we could rename the command in a v2 to what people prefer.

Reply:
Assuming you're referring to this - https://github.com/microsoft/winget-cli/blob/a56372ba05d8f4423cb7cf5be0b5c0bacb445ca3/src/AppInstallerCLICore/Command.cpp#L888

Just clarifying - All that would need to be done to enable telemetry around specific aliases would be to pass the aliasFullName to the telemetry logger instead of the command full name?

Reply:
I think it would be more complicated. We'd probably need a new event, like `Logging::Telemetry().LogCommandAlias(...)` to avoid tripping any data processing we currently have that looks at the full name. Also, we'd need to have `CaseInsensitiveContains` tell us which alias matched. Now that I'm saying this, it seems too complicated and I'm not sure if it is needed..